utility/cert/test_cert.crt

```diff
@@ -1,32 +1,33 @@
-uyCx7Nr5QQXaXsc057wYLdL11sGBJ0Ngd1UGtYPNaqo/GFA8Oq67W2Azwqn1waLD
-dZhiVpJEA1JPWO6l5OYs2kyUDWeahykCtdVqlLRGxwQQ00rmoXHiEkQr3ZrUcfTV
-+A5Ex3YJZAl53ukOX8mBZOnIeQQPCwrgbIWXgyxHSl102HUl27Fxqvt8iy0UpAAo
-ND91WnsCAwEAATANBgkqhkiG9w0BAQsFAAOCAgEAMzayfOnDoItDe501vsj3j0kC
-w+9HI9tRgad+TEhzZlwcE8MVKbVoD+bSiezMS3Ck2Mgu3iLjjDo/TG3l2ufpmdre
-u3eYOj4aMqkm1TQAqGd9TPMsPElspMdIxeFOnm25EOns7SheHyc/iRRPAIvBBm82
-CwxJu4d5EYttsczNxVmpDyRlQyOd6f+MngnQtWoGeU5Nl9CPSN1x9A40wJx4HyTp
-p+bxLKdeygyfVjKj4ebTnQvSZ4cg6YIRgwFPMyjel8gpRBH17OlRcazdpLxo3bXq
-LtLZpEmSIV01savRviT8Qs5vFf/nBrJvTtGUtfqBJ9T97Hi3T3cFCWBelnuuuly6
-W931GJQDjqh+0V1gkULzj5OgMJo0FfCXs659rkvtTbGNMj12BxI6O3tMZIAir7IW
-aAdyFCh7zp8RZcHt2ezQTXwi97MYn1grPEDc7AQ3ilgbgba6Kgtsf9J4Z5HYRBiT
-VdKKtWZgmbKPiRPSWKhvNsVXplHDL9k/5i2Mb6sgNixI8iSrzBVZ4kndwtdjCpmB
-5NHUEbeNiEtLg4pQbvuooMzJiVP2Qgy2KWPbDd4P0jNQy2477zblnc44OZ7vI4oo
-GLZHrpSksO641bEMBLo7lSoZ+OFcs2vzQ36z8FpJ62GPB6JjAQwYGr6KGBNMP6hI
-lU4jaGl70F3o+dQ9fbY=
+MIIFsTCCA5mgAwIBAgIUOxpo3qGWu6SqfqO1MY7OnNiSfRgwDQYJKoZIhvcNAQEL
+BQAwfzELMAkGA1UEBhMCVUExDTALBgNVBAgMBEtpZXYxDTALBgNVBAcMBENpdHkx
+DjAMBgNVBAoMBW9hdHBwMQswCQYDVQQLDAJocTESMBAGA1UEAwwJbG9jYWxob3N0
+MSEwHwYJKoZIhvcNAQkBFhJsZ2Fuenp6b0BnbWFpbC5jb20wHhcNMjMwNjI5MTE0
+MjE3WhcNMjgwNjAyMTE0MjE3WjB/MQswCQYDVQQGEwJVQTENMAsGA1UECAwES2ll
+djENMAsGA1UEBwwEQ2l0eTEOMAwGA1UECgwFb2F0cHAxCzAJBgNVBAsMAmhxMRIw
+EAYDVQQDDAlsb2NhbGhvc3QxITAfBgkqhkiG9w0BCQEWEmxnYW56enpvQGdtYWls
+LmNvbTCCAiIwDQYJKoZIhvcNAQEBBQADggIPADCCAgoCggIBAKWGP7cXGRaEOHYI
+FEXZYKeyyFOa6oRRD2H9uAOo10JFIgcgPforpfmAsueJbB6VhR64YPxIlKPQTnn0
+geePf2xeDtbyBb5PgjK2KqYsO/JKuYeS/Wtb2crGlmooeXy+jIpRUyDulgsx5nFo
+F0za6u58aruuPaHFQlUDDIIkgtZeGZZqeKafF5DMyBwi9JysoikUe8WtZchGtIEI
+LZHMWZQ6wsrENUVJU/AYisCzpDlVChycAvNgcjZdzm7Mlwfw3Z48RwHZYzh21c8e
+cst2Xcduz+StNGdX2VYOBljx1aII4CrnQynfJIMDPM4l9c55p4YAY+QgNir1/qlY
+9CIhyBGmrQKR3l+5tb3/9tFbyYG3IkM/Df+FOr6I2iGvYKFw1ByDhTkp8zpA0Vhm
+a65+WQr2tUOinpF6f81wMTrAXsWkIVwfXonglR0LMtZe8cDF8l/6wz3mWSBAH58o
+J8muldD3rtG4jWRolu3ipLsgseza+UEF2l7HNOe8GC3S9dbBgSdDYHdVBrWDzWqq
+PxhQPDquu1tgM8Kp9cGiw3WYYlaSRANST1jupeTmLNpMlA1nmocpArXVapS0RscE
+ENNK5qFx4hJEK92a1HH01fgORMd2CWQJed7pDl/JgWTpyHkEDwsK4GyFl4MsR0pd
+dNh1Jduxcar7fIstFKQAKDQ/dVp7AgMBAAGjJTAjMCEGA1UdEQQaMBiCCWxvY2Fs
+aG9zdIILdmlydHVhbGhvc3QwDQYJKoZIhvcNAQELBQADggIBABcwU7m1D4R2w2jB
+9wIT3jFSbhl1E2EuGGHkurz/55RfdjQl2rmQJcQRDpURBp6zva+5XGo01uM2DTTY
+QmAzWQnYwO/dICma31wG66ihBbuXD+7kktCHbl690HweQgPorJjnjj2VSc0vpYL0
+o4sspXVOZqrUjAh4PKj4/Ns2epIACOzlXKMdYmnEzgll4NJFylZjUdb5J4psKP8p
+E9DQ20KNRd7wZmtC+9Vrchi7Jsqb+HWvLGhaCy0lY4IabkD8buyRjoiFnRbhEp/N
+bnx+GWSFtbyVMs48w9OJsTyhaxlmZxTA12HltG+v+RQhOXnYASHJ0EKmisO0/uBL
+VR/IwBPW0TWNDAExzOqE35yh3amwmq/Axzqucgb7YZPlJ580knoz2Tr1EwBgwFXq
+UWN3ivOli0s6SLl2vMZGRiZk6VXiEYVUK7zNgk7Pw4ACWhk5wr/9KRmWaQAU6IDk
+bNg6YCEKSjiGi8I1kYXfxitP71VPGYTw2WBBIOOKSkb7a/KY/q6bg0K0MkdZACRv
+CgtTF6zJiNr8Dp59hnQ/ljNDbvGrovjiEqK+4+rCs1vyujwROOSiaLP4/5Qo23so
+XZd12HY2Arb9AcxrGmpeu1+H67sWkWsv82JRVQGpHp1hDL9gCvF22ToUttzKwZ7q
+AX6znd7crITVR82N7yGEZhqk9sHu
```

Comment:
For the unit test, this file now serves as both the certificate for the server and as the trust store for the client.  Normally, these would be separate files.